### ISSUE TITLE
"By Angie Mar November 2019" => 2020-03-01T11:00:00.000Z

### ISSUE THREAD

**Issue by metalwarrior665** [wanasit/chrono#316]
```
var chrono = require('chrono-node');
chrono.parseDate('By Angie Mar November 2019')
```

returns ```2020-03-01T11:00:00.000Z```

Uhh. I know it is really hard to parse human-like dates but this almost forces me to stop using this library.

However, moment parses this correctly 
```2019-11-01T00:00:00+01:00```

**Comment by wanasit** [comment]
This has been fixed in v1.4.5.
